cloudregister/cloudguest_repo_service.py

```diff
@@ -144,10 +143,6 @@ def app():
         )
         if has_plugin_format and not trigger_set:
             trigger_set = True
-            # Python has no sane way to create a detached process
-            subprocess.Popen(
-                ['nohup', '/usr/sbin/switchcloudguestservices'],
-                stdin=subprocess.DEVNULL,
-                stdout=subprocess.DEVNULL,
-                stderr=subprocess.DEVNULL
+            utils.exec_subprocess(
+                ['/usr/sbin/switchcloudguestservices'], disconnect=True
```

Comment:
But this is disappearing ;) in #274 so let's get that merged first than this just goes away.

Reply:
yep done, also dropped the feature for daemon like process calls, as it was used only once in this code that got dropped